package org.thiago.authentication_skeleton.shared.exceptions;

public record ErrorResponseDto(String message) {
}
